skbeam/core/accumulators/tests/test_binned_statistic.py

```diff
@@ -23,51 +24,112 @@ def setup(self):
         colarr = np.arange(colsize)
         rowgrid, colgrid = np.meshgrid(rowarr, colarr, indexing='ij')
         self.rgrid = np.sqrt(rowgrid**2 + colgrid**2)
+        self.phigrid = np.arctan2(colgrid, rowgrid)
```

Comment:
Actually, I think the test should also use `radial_grid` and `angle_grid`. Change this to:
```python
    from skbeam.core.utils import angle_grid, radial_grid
    self.rgrid = radial_grid((0,0),(rowsize, colsize))
    self.phigrid = angle_grid((0,0),(rowsize, colsize))
```

I think once you do this the the previous suggestion of using `angle_grid` and `radial_grid` should work. I fetched your code and made those two changes and it worked for me :-). I think this will help us keep the internal functions self-consistent. What do you think?